src/pybind/mgr/dashboard/frontend/e2e/page-helper.po.ts

```diff
@@ -17,11 +17,76 @@ export abstract class PageHelper {
       .getText();
   }
 
+  static getTableCount() {
+    return $('.datatable-footer-inner.selected-count');
+  }
+
+  static getTitleText() {
+    let title;
+    return browser
+      .wait(() => {
+        title = $('.panel-title');
+        return title.isPresent();
+      })
+      .then(() => title.getText());
+  }
+
+  static getTableCell(content) {
+    return element(by.cssContainingText('.datatable-body-cell-label', content));
+  }
+
+  static getTable() {
+    return element.all(by.css('.datatable-body'));
+  }
+
   static getTabsCount() {
     return $$('.nav.nav-tabs li').count();
   }
 
+  static getFirstTableCellWithText(content) {
+    return element.all(by.cssContainingText('.datatable-body-cell-label', content)).first();
+  }
+
+  // Used for instances where a modal container recieved the click rather than the
+  // desired element
+  static moveClick(object) {
+    return browser
+      .actions()
+      .mouseMove(object)
+      .click();
```

Comment:
This will fix the problem with clicking in `checkbox`.
```suggestion
      .click()
      .perform();
```